wagtail/users/tests/test_admin_views.py

```diff
@@ -1637,7 +1637,7 @@ def test_simple(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "wagtailadmin/generic/listing_results.html")
         # response should not contain page furniture
-        self.assertNotContains(response, "Add a group")
+        self.assertBreadcrumbsNotRendered(response.content)
```

Comment:
Since we now render the filters and button fragments in the base `listing_results.html` template (not just `index_results.html`), the results-only response will contain the "Add a group" string.

This only surfaced now because the groups index results view uses the `listing_results.html` instead of `index_results.html`, despite the full version using `generic/index.html`

See also:
- bea42bdd6aa625b5de64a56068c7b40561c51483
- 81836fc56fc77bacafe642735928335e3951bcb1
- https://github.com/wagtail/wagtail/commit/a0036dd3cb9ab04c52e8be43bfdfe9d2fc43bf9e#diff-28b93323fd36937d1fe777546962e05ccc31a4995a4da38f223eac28e050e939


The users index results view already had the same treatment when it was refactored in https://github.com/wagtail/wagtail/commit/08eda4df2956bf633f560c78d6600ba22fabca94#diff-b49809c33d7d1d10e68f58d2a3d84e99bd50472900e98ea6ff9dc0ed8edbfae2R277-R298